src/components/ContactImportationModal/ContactImportationCard/ImportationInput.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Icon } from 'cozy-ui/react'
+import Importation from '../../../importation'
+
+class ImportationInput extends React.Component {
+  state = {
+    value: ''
+  }
+
+  handleOnChange = event => {
+    const file = event.target.files[0]
+    this.props.fileAction(file)
+    this.setState(state => ({ ...state, value: '' }))
```

Comment:
OK understood. Nice !

Since it is a bit surprising, I think it would be good to have the link to the stackoverflow answer you pointed in the chat :)

Reply:
I put the link into the commit's message.
I'm still not a big fan of comments in the code because they are not historized, while git history (with a `git blame` in the way "but who did this?") lets you know what happened.

RT if you agree
fav if you still want me to add a comment
(or just reply below because we are not on twitter)

Reply:
> I'm still not a big fan of comments in the code because they are not historized,

In what sense are they not historized ? They are historized in the SCM right ?

I understand that we should not have our code littered with stackoverflow answers but I think this one is particularly surprising and would qualify for it. I am not sure I would have `git blame`d this code right away. I think I would have had trouble to understand first.

Reply:
I added a comment